### PR TITLE
#170947424  set up travis ci

### PR DESCRIPTION
**Pull request by blaise82**
#### What does this PR do?
- set up Travis ci
#### Description of Task to be completed?
- write travis.yml file

#### How should this be manually tested?
- clone this repo 
- run npm i

#### What are the relevant pivotal tracker stories?
[fnishes #170947424]

### CODE REVIEW

**Review by hakoemmy — CHANGES_REQUESTED** [review]
Good job @blaise82 but it sounds like you haven't followed [Andela best pratices](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices) while editing your pull request.

There are a couple of issues  that you need to change:

1. Add how can we test your code manually
2. Add a link to the corresponding PR story
3. Use present tense: "write" not, "wrote" or "writing"

**Review by AbdoulNuru — APPROVED** [review]

**Review by izzett222 — APPROVED** [review]
Lgtm

**Comment by blaise82** [comment]
> Good job @blaise82 but it sounds like you haven't followed [Andela best pratices](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices) while editing your pull request.
> 
> There are a couple of issues that you need to change:
> 
> 1. Add how can we test your code manually
> 2. Add a link to the corresponding PR story
> 3. Use present tense: "write" not, "wrote" or "writing"

they are somethings that are still being worked on
I raised this PR just to see if I am connected to Travis ci

**Review by RusimbiPatrick — DISMISSED** [review]
You will need to configure the test coverage reporter. Consider the link below
https://docs.codeclimate.com/docs/travis-ci-test-coverage

**Comment by blaise82** [comment]
> You will need to configure the test coverage reporter. Consider the link below
> https://docs.codeclimate.com/docs/travis-ci-test-coverage

you are right

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - npm install
+services:
+  - postgresql
+before_script:
+  - npm add sequelize-cli
+  - psql -c 'CREATE database bn_test;' -U postgres
+script: 
+  - npm test
+after_success: 
+  - npm run coverage
```
Comment (hakoemmy) [inline .travis.yml]: @blaise82 Would you add a blank line on a new line

**Review by hakoemmy — CHANGES_REQUESTED** [review]
Good job @blaise82 Fix ESLint issue and  I don't see badge in the readme as well.

**Review by hakoemmy — APPROVED** [review]

**Review by RusimbiPatrick — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (kcharles52) [inline .travis.yml]: A blank line at the end shouldn't have space characters. Please make sure you have removed them
```suggestion

```

**Review by kcharles52 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (blaise82) [reply .travis.yml]: @kcharles52 Thank you, spaces removed

**Review by kcharles52 — APPROVED** [review]

**Pull request merged**